uint/tests/uint_tests.rs

```diff
@@ -540,6 +540,19 @@ fn display_uint_zero() {
 	assert_eq!(format!("{}", U256::from(0)), "0");
 }
 
+#[test]
+fn display_uint_format() {
+	let s = "12345678987654321023456789";
```

Comment:
could you also test formatting 0 since we're special casing it?
```suggestion
	let s = U256::from_dec_str("12345678987654321023456789").unwrap();
```